pyg/Pygemony.py

```diff
@@ -45,7 +45,7 @@ def _sanitize_todo_line(self, lines):
     @staticmethod
     def hash_todo(todo_content, file_name):
         m = hashlib.md5()
-        m.update('{}-{}'.format(todo_content, file_name))
+        m.update('{0}-{1}'.format(todo_content, file_name))
```

Comment:
Hey Josh, thanks for taking the time to do this for the project. I agree
with most of it save the change in removing the virtual environment. I have
it specifically there so that if a new requirement is ever added, the build
will fail.

I don't really want an extra requirement and I'd prefer to slowly phase
them out, but it's an important build step in large projects. (although
this is definitely not a large project.)
Im out to lunch at the moment, so I'll review again later tonight.

Thanks,
Ian Clark.

In pyg/Pygemony.py
<https://github.com/GrappigPanda/pygemony/pull/150#discussion_r54336363>:

> @@ -45,7 +45,7 @@ def _sanitize_todo_line(self, lines):
>      @staticmethod
>      def hash_todo(todo_content, file_name):
>          m = hashlib.md5()
> -        m.update('{}-{}'.format(todo_content, file_name))
> +        m.update('{0}-{1}'.format(todo_content, file_name))

Its interesting that the tests passed under python 2.6 before, because they
shouldn't have :)

—
Reply to this email directly or view it on GitHub
<https://github.com/GrappigPanda/pygemony/pull/150/files#r54336363>.